modules/kernel/src/org/apache/axis2/util/Utils.java

```diff
@@ -257,13 +257,20 @@ public static String[] parseRequestURLForServiceAndOperation(String path, String
      * Ex: ..services/foo/bar/Version/getVersion -> foo/bar/Version/getVersion
      * @param path - incoming EPR
      * @param servicePath - Ex: 'services'
+     * @param servicePathValidation - whether to validate the service path or not
      * @return - service/operation part
      */
-    public static String getServiceAndOperationPart(String path, String servicePath) {
+    public static String getServiceAndOperationPart(String path, String servicePath, boolean servicePathValidation) {
```

Comment:
this is a public method, so let's not change the method signature, leave the existing method. overload it and introduce a new method for this purpose.

Reply:
+1 fixed